src/syft/core/common/module.py

```diff
@@ -77,3 +83,81 @@ def cpu(self) -> "Module":
         for _, module in self.modules.items():
             module.cpu()
         return self
+
+    # zero them so we know they are copied
+    def zero_layers(self) -> None:
+        for _, m in self.modules.items():
```

Comment:
NITPICK: Why not use for m in self.modules.values()?

Reply:
Thanks, fixed.